dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/SourceCodeRepoInterface.java

```diff
@@ -431,7 +431,12 @@ public void updateVersionMetadata(String filePath, Version<?> version, Descripto
             fileContent = first.get().getContent();
             LanguageHandlerInterface anInterface = LanguageHandlerFactory.getInterface(type);
             anInterface.parseWorkflowContent(filePath, fileContent, sourceFiles, version);
-            if ((version.getDescription() == null || version.getDescription().isEmpty()) && version.getReference() != null) {
+            // Previously, version has no description
+            boolean noDescription = (version.getDescription() == null || version.getDescription().isEmpty()) && version.getReference() != null;
```

Comment:
It was like this, but is the `version.getReference()` is to ensure it's not a hosted workflow? In any case, I suggest indicating what it's for some how (comment, different variable name).

Reply:
I'm actually not sure

Reply:
I was wondering about this too